bpf/process/types/basic.h

```diff
@@ -2286,6 +2292,13 @@ do_actions(void *ctx, struct selector_action *actions, struct generic_maps *maps
 {
 	bool post = true;
 	__u32 l, i = 0;
+	int zero = 0;
+	struct msg_generic_kprobe *e;
+
+	e = map_lookup_elem(maps->heap, &zero);
+	if (!e)
+		return 0;
+	e->post = true;
```

Comment:
leftover? IIUC this hunk should go away right?

Reply:
Ahh, thanks! Exactly.